js/popup.js

```diff
@@ -1,486 +1,462 @@
 };
 
 // Simple Sign In button clicked
-var onQmSignButtonClicked = function()
-{
-	chrome.tabs.create({ url: "https://app.quantimo.do/analyze" });
+var onQmSignButtonClicked = function () {
+    chrome.tabs.create({url: "https://app.quantimo.do/analyze"});
 };
 
 
-var onQmButtonClicked = function()
-{
-	chrome.tabs.create({ url: "https://app.quantimo.do/analyze" });
+var onQmButtonClicked = function () {
+    chrome.tabs.create({url: "https://app.quantimo.do/analyze"});
 };
-var onCloseButtonClicked = function()
-{
-	window.close();
+var onCloseButtonClicked = function () {
+    window.close();
 };
 
 
 /**
- * Desc: 
+ * Desc:
  */
 
-var onEdtButtonClicked = function()
-{
-	// Create an array of measurements
-	var name = $("#edt-addmeasurement-variable-name").val();
-	var unit = $("#addmeasurement-variable-unit").val();
-	var value = $("#addmeasurement-variable-value").val();
-	var valueCategory = $("#addmeasurement-variable-category").val();
-	var combineOp = $("#combineOperation").val();
-	var datetimeString = $("#addmeasurement-variable-date").val();
-
-	datetimeString = datetimeString.replace("AM","");
-	datetimeString = datetimeString.replace("PM","");
-	
-	var hour = $('#addmeasurement-variable-timeh').val();
-	var min = $('#addmeasurement-variable-timem').val();
-	var ap = $('#addmeasurement-variable-timeap').val();
-	var datetime = new Date(datetimeString);
-	
-	/*
-	alert (datetime) ; 
-	alert (Math.floor(datetime.getTime()  / 1000));
-	/*datetime.setHours(parseInt(hour) + (ap * 12));
-	datetime.setMinutes(min);
-	datetime.setSeconds(0);
-	*/
-
-	if (name == '') {
-		alert("Please enter the variable name."); return;
-	}
-	if (value == '') {
-		alert("Please enter the value."); return;
-	}
-
-	//var variable = getVariableWithName(name);
-	var measurements = 	[
-							{
-								timestamp: 	Math.floor(datetime.getTime()  / 1000), 
-								value: 		value
-							}
-						];
-	//alert ( measurements ) ; 
-	// Add it to a request, payload is what we'll send to QuantiModo
-	var request =		{
-							message: "uploadMeasurements",
-							payload:[
-										{
-											measurements:			measurements,
-											name: 					name,
-											source: 				"QuantiModo",
-											category: 				valueCategory,
-											combinationOperation: 	combineOp,
-											unit:					unit
-										}
-									]
-									
-						};
-
-
-
-	chrome.extension.sendMessage(request, function(responseText) {
-		var response = $.parseJSON(responseText);
-		//alert (response) ; 
-		if(response.success == true)
-		{
-			document.getElementById('addmeasurement-variable-value').value = "";
-			//document.getElementById('edt-addmeasurement-variable-value').value = "";
-			window.close();
-		}
-		else
-		{
-			alert("Adding a measurement failed.");
-			console.log(responseText);
-		}
-	});
-		
-	clearNotifications();
+var onEdtButtonClicked = function () {

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['onEdtButtonClicked' was used before it was defined.](https://www.codacy.com/app/m_3/QuantiModo-Chrome-Extension/file/832637814/issues/source?bid=2779649&fileBranchId=2790922#l157)